src/helpers.js

```diff
@@ -9,11 +9,9 @@ function setImmediatePolyfill(fn) {
 }
 
 // istanbul ignore next
-const {
-  setTimeout,
-  clearTimeout,
-  setImmediate = setImmediatePolyfill,
-} = globalObj
+const clearTimeoutFn = globalObj.clearTimeout
+const setImmediateFn = globalObj.setImmediate || setImmediatePolyfill
```

Comment:
`// istanbul ignore next` probably should have gone above this one

Reply:
lol, I should've waited for the build to finish. Could you make that change please?